app/src/main/res/values/strings.xml

```diff
@@ -42,26 +42,26 @@
     <string name="settings_backup_status_next_backup_usb">Backups will happen automatically when you plug in your USB drive</string>
     <string name="settings_backup_scheduling_title">Backup scheduling</string>
     <string name="settings_backup_app_check_title">Check integrity</string>
-    <string name="settings_backup_app_check_summary">Ensure that backup is working for restore</string>
+    <string name="settings_backup_app_check_summary">Ensure that backup\'s working for restore</string>
     <string name="settings_backup_exclude_apps">Exclude apps</string>
     <string name="settings_backup_now">Backup now</string>
     <string name="settings_category_storage">File backup (beta)</string>
-    <string name="settings_backup_storage_title">Backup my files</string>
+    <string name="settings_backup_storage_title">Backup files</string>
     <string name="settings_backup_files_title">Included files and folders</string>
     <string name="settings_backup_files_summary">None</string>
-    <string name="settings_backup_file_check_title">Check backup integrity</string>
-    <string name="settings_backup_file_check_summary">Make sure my backup is working for restoring files</string>
+    <string name="settings_backup_file_check_title">Check integrity</string>
+    <string name="settings_backup_file_check_summary">Ensure that file backup\'s working for restore</string>
     <string name="settings_backup_recovery_code">Recovery code</string>
-    <string name="settings_backup_recovery_code_summary">Verify existing code or generate a new one</string>
+    <string name="settings_backup_recovery_code_summary">Verify the existing code or generate a new one</string>
     <string name="settings_backup_storage_dialog_title">Beta feature</string>
-    <string name="settings_backup_storage_dialog_message">Backing up files is beta and might not work. Do not rely on it for important data.</string>
+    <string name="settings_backup_storage_dialog_message">File backup\'s still in beta and might not work. Don\'t rely on it for important data.</string>
```

Comment:
"Support for backing up files is a beta feature. Don't rely on this for important data."

Perhaps?